Sources/SDL+JNIExtensions.swift

```diff
@@ -6,7 +6,9 @@
 //  Copyright © 2017 flowkey. All rights reserved.
 //
 
+#if os(Android)
 import JNI
+#endif
```

Comment:
this file would also only work on Android, so can we get away with excluding this file from the build entirely rather than just adding this conditional on the import?